src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -207,7 +201,7 @@
 					x:Load="{x:Bind ViewModel.InstanceViewModel.IsPageTypeRecycleBin, Mode=OneWay, FallbackValue=False}"
 					AccessKey="B"
 					Command="{x:Bind Commands.EmptyRecycleBin}"
-					Content="{x:Bind Commands.EmptyRecycleBin.Icon}"
+					Content="{x:Bind Commands.EmptyRecycleBin.OpacityIcon}"
```

Comment:
Isn't this supposed to be `Icon` now?

Reply:
@cinqmilleans

Reply:
Yes, I'll implement it

Reply:
I think it was a bad merge because @cinqmilleans changed it in his pr